util/syslinux-bios/syslinux/syslinux.cfg

```diff
@@ -1,18 +1,27 @@
 TIMEOUT 50
 DEFAULT vesamenu.c32
+#DEFAULT 0
 MENU TITLE Harvey OS
 PROMPT 0
 ONTIMEOUT 0
 
 LABEL 0
 TEXT HELP
-Boot Harvey kernel
+Boot Harvey kernel as a CPU node
 ENDTEXT
-MENU LABEL Harvey OS
+MENU LABEL Harvey OS (CPU)
 KERNEL mboot.c32
-APPEND ../harvey
+APPEND ../harvey service=cpu nobootprompt=tcp maxcores=1024 fs=10.0.2.2 auth=10.0.2.2 nvram=/boot/nvram nvrlen=512 nvroff=0 acpiirq=1 mouseport=ps2 vgasize=1024x768x24 monitor=vesa"
```

Comment:
you really don't want to hardware fs and auth here long term, I guess it's ok for now.

Reply:
Agreed, need a better way to do this, but I think it's fine for now